lib/src/solver/version_solver.dart

```diff
@@ -185,6 +185,7 @@ class VersionSolver {
     // [incompatibility] is satisfied and we have a conflict.
     if (unsatisfied == null) return #conflict;
 
+    // ignore: missing_whitespace_between_adjacent_strings
```

Comment:
Is there a bug in the lint? Have you filed an issue?

Reply:
mostly for JS compile. 🤷‍♂

Reply:
No bug. In this case, the lint is not valid – because we don't need whitespace here

Reply:
Right, so we should file a bug on the lint.

Reply:
No. The lint is doing the right thing. It makes sure that

```dart
"This is my long sentence error message"
"that's missing a space"
```

is correct. But you get false positives.

Reply:
fixed